linode/manager
# refactor: [M3-6372] - MUI v5 - `Components > PasswordInput`

## Description 📝

- MIUI style migration for `<PasswordInput />`,  `<HideShowText />`,  and `<StrengthIndicator />`

## Major Changes 🔄
- Run `tss-react` codemod on `<StrengthIndicator />`
- Clean up `<PasswordInput />` implementation
- Clean up and rewrite `<HideShowText />` from a class component to a functional component
- Deletes unused `PasswordPanel.tsx`
- Creates Storybook stories for undocumented components

## Preview 📷
![Screenshot 2023-06-01 at 3 16 21 PM](https://github.com/linode/manager/assets/115251059/61ab7ce1-cd29-41ce-a49b-10390bad9a2f)

## How to test 🧪
- Test the password input component
  - You can do this on the Linode Create flow
- Verify e2e tests pass
  - The e2e tests will fail if this component is completely broken
